src/main/resources/gnss_log_2021_02_05_13_20_58.txt

```diff
@@ -0,0 +1,345 @@
+# 
```

Comment:
Could you add a README with basic details on the project, including an explanation of what both the files are for?